opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -111,4 +122,19 @@
      * @return this Span instance, for chaining
      */
     Span setOperationName(String operationName);
+
+    /**
+     * @deprecated use {@link #log(Map)} like this
+     * {@code span.log(Map.of("event", "timeout"))}
```

Comment:
is there a constant file somewhere where we can define "event"? seems equal or more important than some of the trace keys already defined, as this tells the user how to just log a simple string.